setup/aix61/manualBootstrap.txt

```diff
@@ -1,24 +1,60 @@
-ftp://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/wget/wget-1.9.1-3.aix6.1.ppc.rpm
-rpm -i wget-1.9.1-3.aix6.1.ppc.rpm
-ftp://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/unzip/unzip-5.51-2.aix6.1.ppc.rpm
-rpm -i unzip-5.51-2.aix6.1.ppc.rpm
+NOTE: dependencies on AIX seem to make it hard to get the order correct
+so the scripts install-xxx.sh currently need to be run more than
+once and after some small number of runs (2-3) all of the required
+modules will be installed.
+
+
+1) increase the size of the /opt filesystem
+chfs -a size=+1300000 /opt
+
+2) Install required packages
+
+ftp://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/curl/curl-7.44.0-2.aix6.1.ppc.rpm
+rpm -e git (if necessary)
+rpm -e curl (if necessary)
+rpm -ivh curl-7.44.0-2.aix6.1.ppc.rpm
+
+ftp://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/unzip/unzip-6.0-3.aix6.1.ppc.rpm
+rpm -i unzip-6.0-3.aix6.1.ppc.rpm 
 
 http://www.bullfreeware.com/affichage.php?id=2378 - gcc-c++ + all dependencies
 gcc-c++-4.8.5-1.aix6.1.ppc.rpm-with-deps.zip
 unzip gcc-c++-4.8.5-1.aix6.1.ppc.rpm-with-deps.zip
 sh install-gcc-rpm.sh
 
-http://www.bullfreeware.com/affichage.php?id=2080  - git + all dependencies
-git-1.8.3.1-3.aix6.1.ppc.rpm-with-deps.zip
-unzip git-1.8.3.1-3.aix6.1.ppc.rpm-with-deps.zip
-sh install-git-tools-rpm.sh
+curl --insecure https://public.dhe.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/git/git-2.8.1-1.aix6.1.ppc.rpm >git-2.8.1-1.aix6.1.ppc.rpm
+rpm -ivh git-2.8.1-1.aix6.1.ppc.rpm
+
+Update openssl and openssh
+
+get  OpenSSH_6.0.0.6202.tar.Z and openssl-1.0.2.800.tar.Z from:
+https://www-01.ibm.com/marketing/iwm/iwm/web/reg/pick.do?source=aixbp&lang=en_US
+copy into directory fixup
+cd fixup
+uncompress OpenSSH_6.0.0.6202.tar.Z
+uncompress openssl-1.0.2.800.tar.Z
+tar -xvf OpenSSH_6.0.0.6202.tar.Z
+tar -xvf openssl-1.0.2.800.tar.Z
+copy contents of OpenSSH_6.0.0.6202  openssl-1.0.2.800 to fixup dir.
+inutoc .
+
+emgr -r -L IV80743m9a  (if necessary)
+emgr -r -L IV83169m9a  (if necessary)
+
+installp -Y -qaXgd . openssl openssh 
+
+curl --insecure https://public.dhe.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/gettext/gettext-
+0.19.7-1.aix6.1.ppc.rpm >gettext-0.19.7-1.aix6.1.ppc.rpm
+rpm -e gettext 
+rpm -hUv gettext-0.19.7-1.aix6.1.ppc.rpm
+
 
 Java7r1_64.jre.7.1.0.200.tar.gz
 gunzip Java7r1_64.jre.7.1.0.200.tar.gz
 tar -xvf Java7r1_64.jre.7.1.0.200.tar
 installp -agXYd . Java71_64.jre 2>&1 | tee installp.log
 
-wget http://www.oss4aix.org/download/everything/RPMS/make-3.82-1.aix5.3.ppc.rpm
+curl --insecure http://www.oss4aix.org/download/everything/RPMS/make-3.82-1.aix5.3.ppc.rpm -O
```

Comment:
Without the --insecure it complains about the cert and does not download.  There is probably a way to specify more certificates as well but not sure what that is without additional investigation.